apps/docs/package.json

```diff
@@ -12,8 +12,8 @@
     "@shopware-pwa/api-client": "workspace:*",
     "@shopware-pwa/composables-next": "workspace:*",
     "@shopware-pwa/helpers-next": "workspace:*",
-    "vitepress": "1.0.0-alpha.64",
-    "vitepress-shopware-docs": "^0.1.9",
+    "vitepress": "1.0.0-alpha.65",
+    "vitepress-shopware-docs": "0.3.0-beta.38",
```

Comment:
Prepared fix for audit issue: https://github.com/shopware/developer-documentation-vitepress/pull/34

Reply:
Once this is approved, your top navigation bar will disappear (either way, it's broken at the moment). Please, also add this config to your `.vitepress/config.ts:themeConfig.nav`:

```js
    {
        link: 'https://developer.shopware.com/docs/guides/plugins/apps',
        text: 'Apps',
    },
    {
        link: 'https://developer.shopware.com/docs/guides/plugins/themes',
        text: 'Themes',
    },
    {
        link: 'https://shopware-frontends-docs.vercel.app/',
        text: 'Frontends',
        repo: 'shopware/frontends',
    },
    {
        link: 'https://developer.shopware.com/docs/guides/integrations-api',
        text: 'Integrations',
    },
    {
        text: "Resources",
        activeMatch: `^/(api)`,
        items: [
            {
                text: "HTTP APIs",
                items: [
                    {
                        text: "Store API",
                        link: 'https://shopware.stoplight.io/docs/store-api/',
                        repo: 'shopware/store-api-reference',
                    },
                    {
                        text: "Admin API",
                        link: 'https://shopware.stoplight.io/docs/admin-api/',
                        repo: 'shopware/admin-api-reference',
                    }
                ]
            },
            {
                text: "Administration",
                items: [
                    {
                        text: "Admin Extension SDK",
                        link: 'https://shopware.github.io/admin-extension-sdk/',
                        repo: 'shopware/admin-extension-sdk',
                    },
                    {
                        text: "Meteor Icon Kit",
                        link: 'https://shopware.github.io/meteor-icon-kit/',
                        repo: 'shopware/meteor-icon-kit',
                    },
                    {
                        text: "Component Library",
                        link: 'https://shopware.github.io/meteor-component-library/',
                        repo: 'shopware/meteor-component-library',
                    }
                ]
            },
            {
                text: "Learning",
                items: [
                    {
                        text: "Academy",
                        link: "https://academy.shopware.com/",
                    },
                    {
                        text: "YouTube",
                        link: "https://www.youtube.com/user/shopwareAG"
                    }
                ]
            }
        ]
    }
```

Reply:
Done :+1: